store/migration/dev/20221118000000##flexible_policy.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TYPE resource_type AS ENUM ('WORKSPACE', 'ENVIRONMENT', 'PROJECT', 'INSTANCE', 'DATABASE');
+
+DROP INDEX idx_policy_environment_id;
```

Comment:
I am wondering will "ALTER TABLE policy DROP COLUMN environment_id" alone be sufficient.

Reply:
Trying to be correct and readable than optimal. It's better to state what to drop at atomic level.